wls-gui-wahllokalsystem/tests/stores/vorfaelleundvorkommnisseStore.spec.ts

```diff
@@ -0,0 +1,142 @@
+import { createPinia, setActivePinia } from "pinia";
+import { afterEach, beforeEach, describe, expect, it, vi } from "vitest";
+
+import { useUserStore } from "@/stores/user";
+import { useEreignisStore } from "@/stores/vorfaelleundvorkommnisseStore.ts";
+import User from "@/types/User";
+import { WahlbezirkEreignisseBuilder } from "@/types/vorfaelleundvorkommnisse/WahlbezirkEreignisse.ts";
+
+const mockDefinitions = vi.hoisted(() => ({
+  getEreignisse: vi.fn(),
+  saveEreignisse: vi.fn(),
+}));
+
+vi.mock("@/composables/vorfaelleundvorkommnisse/ereignisService", () => ({
+  useEreignisService: () => ({
+    getEreignisse: mockDefinitions.getEreignisse,
+    saveEreignisse: mockDefinitions.saveEreignisse,
+  }),
+}));
+
+const mockedNow = new Date();
+
+describe("vorfaelleundvorkommnisseStore.ts", () => {
+  let unitUnderTest: ReturnType<typeof useEreignisStore>;
+
+  beforeEach(() => {
+    // creates a fresh pinia and makes it active
+    // so it's automatically picked up by any useStore() call
+    // without having to pass it to it: `useStore(pinia)`
+    setActivePinia(createPinia());
+    vi.useFakeTimers({
+      now: mockedNow,
+    });
+    unitUnderTest = useEreignisStore();
+  });
+
+  afterEach(() => {
+    vi.clearAllMocks();
+    vi.useRealTimers();
+  });
+
+  describe("loadEreignisse", () => {
+    it("should_setWahlbezirkEreignisse_when_userHasWahlbezirkID", async () => {
```

Comment:
```suggestion
  describe("loadEreignisse", () => {
    it("should_loadWahlbezirkEreignisse_when_userHasWahlbezirkID", async () => {
```
load und set passt denke ich nicht zusammen